### PR TITLE
Rewrite the README to cover main endpoints

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/tHxgpX1j/229-rewrite-the-readme-for-email-alert-frontend-so-its-not-wrong

Previously the README was out-of-date, stating the only purpose of this app
was to render (legacy) 'email_alert_signup' pages. This rewrites the README
to explain the existence of two signup routes, as well as other routes for
verifying and managing subscriptions.

We do not need a picture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,67 @@
...
+"tags": { "countries": ["foreign-travel-advice/canada"] }
 ```
 
-## Content Item
+```
+"links": { "countries": ["f402b8de-2e99-4ff3-949f-31fe65796cae"] }
+```
 
-The [schema](https://github.com/alphagov/govuk-content-schemas/tree/master/formats/email_alert_signup) sets out the fields required for the Content Item. The only two fields required in the details hash are `summary` and `tags`:
+## Running the test suite
```
Comment (issyl0) [inline README.md:59]: Is this "running the test suite" section not really obvious?

Comment (benthorner) [reply README.md:59]: Apparently it varies: https://github.com/alphagov/content-publisher#running-the-test-suite. It does kind of help to reassure people they've run all the tests, I suppose.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,66 @@
...
 
-## Live examples
+This app provides two routes for signing up to email:
 
-- [gov.uk/government/policies/academies-and-free-schools/email-signup](https://www.gov.uk/government/policies/academies-and-free-schools/email-signup)
+- A legacy `/:base_path/email-signup` route [[example](https://www.gov.uk/foreign-travel-advice/canada/email-signup)].
+
+Each of these routes corresponds to a content item with an [`email_alert_signup` schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/email_alert_signup.jsonnet). At the time of writing, `/foreign-travel-advice/*` and `/service-manual/*` still use this route.
+
+- A `/email-signup?link=/:base_path` route [[example](https://www.gov.uk/email-signup/?topic=/money)].
```
Comment (chao-xian) [inline README.md:15]: I might move this up above the legacy examples, and have Legacy as a sub-heading

File: README.md [hunk 3]
```diff
@@ -1,40 +1,66 @@
...
-$ bundle exec rake
+tags: { topics: { any: ["business-tax/vat"] } }
 ```
 
-## Content Item
+```
+links: { topics: { any: ["1ddaaacb-0981-4abc-a532-a5111f2bea6b"] } }
+```
+
+### Tags and links (again!)
```
Comment (chao-xian) [inline README.md:47]: Think this heading could be clearer as "Legacy tags and links"?

File: README.md (hunk 2 above)
Comment (benthorner) [reply README.md:11]: Sure, although I've gone with **bold** to avoid too many headings.

File: README.md (hunk 3 above)
Comment (benthorner) [reply README.md]: Sure, updated 👍

**Pull request merged**